recipe/meta.yaml

```diff
@@ -1,16 +1,20 @@
 {% set name = "metis" %}
-{% set version = "5.1.0" %}
+{% set version = "5.1.1" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
-  fn: {{ name }}-{{ version }}.tar.gz
-  url: http://glaros.dtc.umn.edu/gkhome/fetch/sw/metis/{{ name }}-{{ version }}.tar.gz
-  sha256: 76faebe03f6c963127dbb73c13eab58c9a3faeae48779f049066a21c087c5db2
-  patches:
-    - vc14.patch
+  - url: https://github.com/KarypisLab/METIS/archive/v{{ version }}-DistDGL-v0.5.tar.gz
+    sha256: cedf0b32d32a8496bac7eb078b2b8260fb00ddb8d50c27e4082968a01bc33331
+    patches:
+      - patches/0001.patch
+      - patches/0002.patch
+
+  - url: https://github.com/KarypisLab/GKlib/archive/3eabb216ac97e11ce7e7a9b90f4c90778d9e7c18.zip
+    sha256: e75693a69957445171843a767430db7270de57fa431a415cc3f8e9ed472700e3
+    folder: GKlib
```

Comment:
Need to package the license for this dependency

Reply:
Can this be a conda package? Why is this vendored in?

Reply:
There is a package for it already: https://github.com/conda-forge/gklib-feedstock

I am just not sure how in sync it is with this one (even if its version is 5.1.1). Upstream folks have not been very easy to work with in the last few years.

Reply:
Yes, I would really like to improve the feedstock for this package (and gklib) but I can't get everything perfect all at once, so I would like to get things improved by getting some changed merged in upstream for the build system. I will get the license packaged.

Reply:
@isuruf license packaged!